contracts/utils/Create2.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @title Create2
+ *
+ * @dev Utility library that focus on the use of CREATE2 EVM opcode for
+ * contracts deployment. It also provides a function to precompute the address
+ * where the smart contracts with the specified salt and bytecode would be
+ * deployed.
+ */
+library Create2 {
+
+    // Event triggered when a contract is deployed
+    event Create2Deployed(address addr, bytes32 salt);
+
+    /**
+     * @dev Deploy contract with CREATE2
+     * @param salt The salt used to the contract address computation
+     * @param code The bytecode of of the contract to be deployed
+     */
+    function deploy(bytes32 salt, bytes memory code) internal {
+        address addr = _deploy(salt, code);
```

Comment:
Maybe you can use `deployAddress` to replace `addr`, it is shorter, but it will be more difficult to understand.